src/status_im/utils/gfycat/core.cljs

```diff
@@ -9,11 +9,20 @@
   [gen vector]
   (str/capitalize (rnd/seeded-rand-nth gen vector)))
 
+(defn- build-gfy
+  [public-key]
+  (let [gen               (rnd/rand-gen public-key)
+        first-adjective   (pick-random gen adjectives/data)
+        second-adjective  (pick-random gen adjectives/data)
+        animal            (pick-random gen animals/data)]
+    (str first-adjective " " second-adjective " " animal)))
+
+(defn- unknown-gfy "Unknown")
```

Comment:
should be `def`

Reply:
Yep for some reason my build commands was doing nothing. So I actually did no see the error. Now it's working after the change mentioned above. Thanks !